src/View/Form/ContextFactory.php

```diff
@@ -59,44 +58,44 @@ public static function createWithDefaults(array $providers = [])
         $providers = [
             [
                 'type' => 'orm',
-                'callable' => function ($request, $data) {
+                'callable' => function ($data) {
```

Comment:
This is a potentially breaking change for contexts defined in userland code. We should continue passing the request into factory functions just in case someone is using it.